tests/phpunit/tests/comment/wpHandleCommentSubmission.php

```diff
@@ -882,4 +882,136 @@ public function test_comments_flood_user_is_admin() {
 		$this->assertNotWPError( $second_comment );
 		$this->assertEquals( self::$post->ID, $second_comment->comment_post_ID );
 	}
+
+	/**
+	 * Tests that wp_handle_comment_submission() only allows replying to
+	 * an approved parent comment.
+	 *
+	 * @ticket 53962
+	 *
+	 * @dataProvider data_should_only_allow_replying_to_an_approved_parent_comment
+	 *
+	 * @param int $approved Whether the parent comment is approved.
+	 */
+	public function test_should_only_allow_replying_to_an_approved_parent_comment( $approved ) {
+		$user = self::factory()->user->create_and_get(
+			array(
+				'user_url' => 'http://user.example.org',
+			)
+		);
+
```

Comment:
Rather than create four users, is it possible to move this to the shared fixtures?

Not a blocker tho.

Reply:
Good point! Done in [fe1f2cf](https://github.com/WordPress/wordpress-develop/commit/fe1f2cf8a57600ec7bf6b03bb308148c1f6c978a).